lib/vanagon/platform.rb

```diff
@@ -237,8 +237,12 @@ def [](key)
     # @param target_repo [String] optional repo target for built packages defined
     #   at the project level
     # @return [String] relative path to where packages should be output to
-    def output_dir(target_repo = "")
-      @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
+    def output_dir(target_repo = nil)
+      if target_repo.nil?
+        @output_dir ||= "#{@os_name}-#{@os_version}-#{@architecture}"
+      else
+        @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
```

Comment:
If, say, a project hasn't updated the beaker version they are using, they could still update to this change of vanagon, keep everything else the same, and it would all still work just as it always has.